keps/sig-node/4176-cpumanager-spread-cpus-preferred-policy/README.md

```diff
@@ -365,76 +368,16 @@ https://storage.googleapis.com/k8s-triage/index.html
 We expect no non-infra related flakes in the last month as a GA graduation criteria.
 -->
 
-- <test>: <link to test coverage>
+No new e2e tests for kubelet are planned.
```

Comment:
I'm afraid I have to disagree here: we will need e2e test coverage for the new feature, which almost certainly mean new e2e tests. They can deferred for beta and beyond though (not alpha blocker, even though the sooner the better)

Reply:
Got it. I will make the e2e test available in alpha version. Then we should cover all the tests for this feature.